src/components/BatchVoting/VotingOptions/VoteSummary.tsx

```
@@ -0,0 +1,111 @@
+
+	const { resolvedTheme: theme } = useTheme();
+
+	const bnToIntBalance = function (bn: BN): number {
+		return Number(formatBnBalance(bn, { numberAfterComma: 6, withThousandDelimitor: false }, network));
+	};
+
+	const ayeVotesNumber = ayesNum === undefined ? bnToIntBalance(ayeVotes || ZERO) : Number(ayesNum);
+	const totalVotesNumber = ayesNum === undefined || naysNum === undefined ? bnToIntBalance(ayeVotes?.add(nayVotes || ZERO) || ZERO) : Number(ayesNum) + Number(naysNum);
+	const ayePercent = (ayeVotesNumber / totalVotesNumber) * 100;
+	const nayPercent = 100 - ayePercent;
+	const isAyeNaN = isNaN(ayePercent);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)